src/code/audio_load.c

```diff
@@ -124,24 +221,24 @@ void func_800E1618(s32 arg0) {
             func_800DE238(temp_s0->ramAddr, gAudioContext.unk_288C);
             temp_s0->size = gAudioContext.unk_288C;
             temp_s0->devAddr = 0U;
-            temp_s0->unk_08 = 0;
-            temp_s0->unk_0C = 0;
-            temp_s0->unk_0E = 0;
+            temp_s0->sizeUnused = 0;
+            temp_s0->unused2 = 0;
+            temp_s0->ttl = 0;
             gAudioContext.sampleDmaReqCnt++;
         }
     }
 
-    for (i = gAudioContext.unk_2624; i < gAudioContext.sampleDmaReqCnt; i++) {
-        gAudioContext.unk_272C[i - gAudioContext.unk_2624] = i;
-        gAudioContext.sampleDmaReqs[i].unk_0D = i - gAudioContext.unk_2624;
+    for (i = gAudioContext.sampleDmaListSize1; (u32)i < gAudioContext.sampleDmaReqCnt; i++) {
+        gAudioContext.sampleDmaReuseQueue2[i - gAudioContext.sampleDmaListSize1] = i;
+        gAudioContext.sampleDmaReqs[i].reuseIndex = i - gAudioContext.sampleDmaListSize1;
     }
 
     for (i = gAudioContext.sampleDmaReqCnt; i < 0x100; i++) {
```

Comment:
On the one hand yes, on the other it's a somewhat magical size that has to be 0x100, because the code relies on u8 overflow. So I think using ARRAY_COUNT might make it less clear